react/components/OrganizationUsersTable.tsx

```diff
@@ -80,7 +84,11 @@ const OrganizationUsersTable: FunctionComponent<Props> = ({
   }
 
   const canEdit = isAdmin || permissions.includes('add-users-organization')
-  const canEditSales = isAdmin || permissions.includes('add-sales-users-all')
+  const canEditSales =
+    isAdmin ||
+    permissions.includes('add-sales-users-all') ||
+    permissions.includes('add-sales-users-current')
```

Comment:
Hmm - if I understand correctly, this will allow Sales Managers to add and edit Sales Admins, which I don't think we want.

Reply:
This flag is used to handle all the sales action in general, however, to control that, I'm using the role directly, so in this way, I enable the "Edit mode" e Sales Admin: 
<img width="527" alt="image" src="https://user-images.githubusercontent.com/5812946/169825506-d3cb4b58-f384-4ab5-b1a0-9aba318f8c45.png">

Reply:
The logic in `handleRowClick` looks good, but the logic to determine whether a row has the "disabled" CSS class needs to be updated so that it matches. When I'm a Sales Manager, for example, the Sales Admin rows still look like they're clickable, even though nothing happens when I click: 
<img width="865" alt="image" src="https://user-images.githubusercontent.com/6306265/170124914-e1134d76-c63a-4749-9bd3-bf8be937c3fa.png">